guides/client/form-bindings.md

```diff
@@ -127,6 +127,9 @@ def input(%{field: %Phoenix.HTML.FormField{} = field} = assigns) do
 
 Now only errors for fields that were interacted with are shown.
 
+To disable sending of `_unused` parameters, you can annotate individual inputs or the whole form with
```

Comment:
@woylie might be useful for resolving a portion of https://github.com/woylie/flop_phoenix/issues/250